### PR TITLE
Feature christian

### PR DESCRIPTION
**Pull request by ChristianBloemhof20**

### CODE REVIEW

File: client/src/components/Checkout/Details.js
Comment (d-mooers) [inline client/src/components/Checkout/Details.js:25]: Is this the same exact component from /item/Details? You can just import it from there

File: client/src/components/Checkout/checkout.js
Comment (d-mooers) [inline client/src/components/Checkout/checkout.js]: Ideally goBack() would put them back in the page they were before visiting /checkout?

File: client/src/components/Item/index.js
Comment (d-mooers) [inline client/src/components/Item/index.js]: Why did you remove the whole functionality to delete an item?

File: client/src/components/Item/index.js
Comment (d-mooers) [inline client/src/components/Item/index.js]: ?????

File: client/src/components/Item/index.js
Comment (d-mooers) [inline client/src/components/Item/index.js]: Deleted? ...why?

File: client/src/components/QuestionsPage/Details.js
Comment (d-mooers) [inline client/src/components/QuestionsPage/Details.js:1]: You can just import this component from the Item folder rather than make a new file for it.

File: client/src/components/QuestionsPage/QuestionsPage.js
Comment (d-mooers) [inline client/src/components/QuestionsPage/QuestionsPage.js:16]: import Details from "../Item/Details"

File: node_modules/.package-lock.json
Comment (d-mooers) [inline node_modules/.package-lock.json:1]: there shouldn't be a node_modules folder in the top level directory.  Did you run an npm install command while you were outside of the client directory? Please remove

File: client/src/components/RegisterPage/Registration.js
Comment (d-mooers) [inline client/src/components/RegisterPage/Registration.js]: Payload for registering is just 
{
username,
email,
password
} no need for auth.

**Review by d-mooers — CHANGES_REQUESTED** [review]
Lets wait until after sprint 2 to merge these changes.  Theres a lot to go over and I would rather do that during a meeting than over discord.

**Review by slaterOwen — COMMENTED** [review]
Everything looks good to me! Good job.

File: client/src/components/QuestionsPage/Details.js
Comment (ChristianBloemhof20) [reply client/src/components/QuestionsPage/Details.js:1]: There are slight changes made to the file which is why I do not import that.  I'll place it into the shared folder since 2 of my pages use it

File: client/src/components/Checkout/checkout.js
Comment (ChaseVZ) [inline client/src/components/Checkout/checkout.js:117]: I like your use of caps to distinguish this object. I might consider updating my code to replicate this.

File: client/src/components/RegisterPage/Registration.js
Comment (ChaseVZ) [inline client/src/components/RegisterPage/Registration.js:111]: change to props.history.push() for consistency

File: client/src/components/RegisterPage/Registration.js
Comment (ChaseVZ) [inline client/src/components/RegisterPage/Registration.js:89]: You can simplify all of these functions to be of form
setUser({ ...user, email: value});
This would be the example for email

File: client/src/components/RegisterPage/Registration.js
Comment (ChaseVZ) [inline client/src/components/RegisterPage/Registration.js]: you'll want to change this post call to take in an object that is just the username, password, and email. Currently, the secondPass is being saved in our database because you are sending the whole temp object (for some reason that version of your code isn't showing up here but you know what im referring to)

**Review by ChaseVZ — COMMENTED** [review]
Code looks good. Looks clean, again I would just simplify your setter functions for your tempUser in registration.js and then make sure you update the POST /user call to not include secondPass.

**Comment by ChristianBloemhof20** [comment]
There's a lot of files here from the last sprint, I don't know if we're not gonna incorporate them or not but you can see the tests I created.

**Pull request closed without merging**
